src/store/getters.js

```diff
@@ -53,5 +53,40 @@ export default {
     }, {})
 
     return Object.assign({}, baseAssets, customAssets)
+  },
+  networkAssets (state) {
+    const { enabledAssets, activeNetwork, activeWalletId } = state
+    return enabledAssets[activeNetwork][activeWalletId]
+  },
+  orderedBalances (state, getters) {
+    const { enabledAssets, activeNetwork, activeWalletId } = state
+    const { networkWalletBalances } = getters
+    const assets = enabledAssets[activeNetwork][activeWalletId]
+    return Object.entries(networkWalletBalances)
+      .filter(([asset]) => assets.includes(asset))
+      .sort(([assetA], [assetB]) => {
+        return assets.indexOf(assetA) - assets.indexOf(assetB)
+      })
+  },
+  networkWalletBalances (state) {
+    const { balances, activeNetwork, activeWalletId } = state
+    if (!balances[activeNetwork]) return false
+    if (!balances[activeNetwork][activeWalletId]) return false
+
+    return balances[activeNetwork][activeWalletId]
```

Comment:
something i found recently. you can remove the checks for undefined and instead do
`return balances[activeNetwork]?.[activeWalletId]`